### PR TITLE
don't error out if undeploy is called > 1 time

### LINKED ISSUE #365: Don't throw traceback if undeploying something already deployed
```
robertso@robertso-mac02:/tmp/wat$ mlt3 undeploy
Error from server (NotFound): error when deleting "k8s/tfjob.yaml": tfjobs.kubeflow.org "wat-9b3125e3-df1e-4299-b583-e3f5fe742cbb" not found
Traceback (most recent call last):
  File "/Users/robertso/Nervana/mlt/.venv3/bin/mlt", line 11, in <module>
    load_entry_point('mlt==0.2.0+30.g10f7791', 'console_scripts', 'mlt')()
  File "/Users/robertso/Nervana/mlt/.venv3/lib/python3.6/site-packages/mlt/main.py", line 189, in main
    run_command(args)
  File "/Users/robertso/Nervana/mlt/.venv3/lib/python3.6/site-packages/mlt/main.py", line 128, in run_command
    CommandClass(args).action()
  File "/Users/robertso/Nervana/mlt/.venv3/lib/python3.6/site-packages/mlt/commands/undeploy.py", line 54, in action
    raise_on_failure=True)
  File "/Users/robertso/Nervana/mlt/.venv3/lib/python3.6/site-packages/mlt/utils/process_helpers.py", line 30, in run
    raise e
  File "/Users/robertso/Nervana/mlt/.venv3/lib/python3.6/site-packages/mlt/utils/process_helpers.py", line 27, in run
    output = check_output(command, cwd=cwd).decode("utf-8")
  File "/usr/local/Cellar/python/3.6.4_3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/local/Cellar/python/3.6.4_3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['kubectl', '--namespace', 'robertso', 'delete', '-f', 'k8s']' returned non-zero exit status 1.
```
We show the error but also raise it. We should `ignore_errors` here.

### PR DESCRIPTION
**Pull request by s1113950**
fixes #365 

If you aren't done, please add the `wip` label until you're ready for review.

If you are adding a new template or mlt command/flag ensure that is is documented in the [docs/features.md](https://github.com/IntelAI/mlt/tree/master/docs/features.md) file.

## Reviewer Checklist

 - [ ] Do you understand it?
 - [ ] Is it correct?
 - [ ] Is it safe?
 - [ ] Is it legally compliant?
 - [ ] Is it robust?
 - [ ] Is it simple?
 - [ ] Is it tested?
 - [ ] Is it documented?
 - [ ] Is the style consistent?

See [the reviewer guide](docs/reviews.md) for more detail on each check.

### CODE REVIEW

**Comment by s1113950** [comment]
Gotta decide if this is good enough:
```
robertso@robertso-mac02:/tmp/wat$ mlt3 undeploy
Error from server (NotFound): error when stopping "k8s/job.yaml": jobs.batch "wat-32f3c6fc-8431-45f6-b3eb-af5c9971d133" not found
robertso@robertso-mac02:/tmp/wat$ echo $?
0
```
or if we don't want anything to print

File: mlt/commands/undeploy.py [hunk 1]
```diff
@@ -49,9 +49,11 @@ def action(self):
         else:
             # don't delete namespace here because we could be using a
             # user-provided namespace
+            # can't think of a reason to want to error out mlt if user
+            # has undeployed multiple times so ignoring this case for now
             process_helpers.run(
                 ["kubectl", "--namespace", namespace, "delete", "-f", "k8s"],
-                raise_on_failure=True)
+                ignore_failure=True)
```
Comment (dmsuehir) [inline mlt/commands/undeploy.py]: Is it possible that we will get some other error besides it being undeployed multiple times, in which case the user would want to know?  Should we actually be checking the error message and just filter out the known error when it's already undeployed?

Comment (s1113950) [reply mlt/commands/undeploy.py]: I was chatting with @Ajay191191 about this. We came up with these scenarios:
1. User changes something in the kubernetes yaml between create and delete
2. User changes namespaces in between create and delete

@Ajay191191 did I miss any?

Comment (Ajay191191) [reply mlt/commands/undeploy.py]: Cluster is not reachable?

Comment (s1113950) [reply mlt/commands/undeploy.py]: Even so, if something could be created then what's the chance that suddenly the cluster is unreachable? 🤷‍♂️ maybe we should still take all these into account

File: mlt/utils/process_helpers.py [hunk 2]
```diff
@@ -22,14 +22,22 @@
...
+def run(command, cwd=None, raise_on_failure=False, ignore_failure=False):
+    """wrapper around check_output call
+       raise_on_failure: mlt will raise error if output call failed
+       ignore_failure: mlt won't even print out any error that occurred
+                       (useful for mlt undeploy call, in case undeploy
+                        was called multiple times for some reason)
+    """
+    output = ''
     try:
         output = check_output(command, cwd=cwd).decode("utf-8")
```
Comment (ashahba) [inline mlt/utils/process_helpers.py]: I'd read and parse the output and only `ignore` specific well known errors here.

**Comment by s1113950** [comment]
We'll go with a change in #394 instead of this one

**Pull request closed without merging**